changehc/delphi_changehc/config.py

```diff
@@ -0,0 +1,57 @@
+"""
+This file contains configuration variables used to generate the CHC signal.
+
+Author: Aaron Rumack
+Created: 2020-10-14
+"""
+
+from datetime import datetime, timedelta
+import numpy as np
+
+
+class Config:
+    """Static configuration variables.
+    """
+
+    ## dates
+    FIRST_DATA_DATE = datetime(2020, 1, 1)
+
+    # number of days training needs to produce estimate
+    # (one day needed for smoother to produce values)
+    BURN_IN_PERIOD = timedelta(days=1)
+
+    # shift dates forward for labeling purposes
+    DAY_SHIFT = timedelta(days=1)
+
+    ## data columns
+    COVID_COL = "COVID"
+    DENOM_COL = "Denominator"
+    COUNT_COLS = ["COVID"] + ["Denominator"]
+    DATE_COL = "date"
+    GEO_COL = "fips"
+    ID_COLS = [DATE_COL] + [GEO_COL]
+    FILT_COLS = ID_COLS + COUNT_COLS
+    DENOM_COLS = [GEO_COL, DATE_COL, DENOM_COL]
+    COVID_COLS = [GEO_COL, DATE_COL, COVID_COL]
+    DENOM_DTYPES = {"date": str, "Denominator": str, "fips": str}
+    COVID_DTYPES = {"date": str, "COVID": str, "fips": str}
+
+    SMOOTHER_BANDWIDTH = 100  # bandwidth for the linear left Gaussian filter
+    MIN_DEN = 100  # number of total visits needed to produce a sensor
+    MAX_BACKFILL_WINDOW = (
+        7  # maximum number of days used to average a backfill correction
+    )
+    MIN_CUM_VISITS = 500  # need to observe at least 500 counts before averaging
+
+
+class Constants:
+    # number of counties in usa, including megacounties
```

Comment:
can we add a docstring here on what these constants are for, especially since there's also a constants.py file?

Reply:
this + linter and should be good to go.